osu.Game/GameModes/Menu/ButtonSystem.cs

```diff
@@ -285,37 +290,35 @@ public override void Load()
             {
                 base.Load();
 
-                Add(logoBounceContainer = new AutoSizeContainer());
-
-                logo = new Sprite(Game.Textures.Get(@"menu-osu"))
-                {
-                    Anchor = Anchor.Centre,
-                    Origin = Anchor.Centre,
-                };
-
-                logoBounceContainer.Add(logo);
-
                 Sprite ripple = new Sprite(Game.Textures.Get(@"menu-osu"))
                 {
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                     Alpha = 0.4f
                 };
-
-                logoBounceContainer.Add(ripple);
-
                 ripple.ScaleTo(1.1f, 500);
                 ripple.FadeOut(500);
                 ripple.Loop(300);
 
-                logoBounceContainer.Add(vis = new MenuVisualisation()
+                Add(logoBounceContainer = new AutoSizeContainer()

```

Comment:
Yeah. I think by using `Children =` for the initial tree initialisation process it will make it easier to locate tree initialisations and possibly help with automatic migration to OML when that comes around.